models/lido_liquidity/lido_liquidity.sql

```diff
@@ -0,0 +1,39 @@
+{{ config(
+        alias ='lido_liquidity',
```

Comment:
similar idea on this spell, the output would be: `lido_liquidity.lido_liquidity` -- i'd recommend a cleaner name

Reply:
maybe `lido.liquidity`?

Reply:
GM @gregshestakovlido,
since `model/lido/` already existed, I will try to refactor the path for spells in this PR to under `models/lido`.